### PR TITLE
General package maintenance. Updated STUN servers list.

### PR DESCRIPTION
**Pull request by Renelvon**
### Motivation

My interest with `pystun` stems from the OpenBazaar project; we are using it to address some of our NAT issues. Unfortunately, it appears some of the package's builtin STUN servers have fallen out of service and this is causing problems. Instead of doing a minor patch, I decided to groom the package for good and bring various aspects (packaging, setup, testing, etc) up to date. I intend to work on it much more.
### Changelog
- Install Continuous Integration via Travis CI.
- Install static analysis (and PEP8 checking) via [prospector](https://pypi.python.org/pypi/prospector).
- Install a unit test infrastructure via nose. Unit test argument parsing.
- Add proper LICENSE and CHANGELOG files.
- Replace deprecated `optparse` module with `argparse`. This drops out-of-the box support for Python versions older than 2.7.
- Port `print` statements to Python 3 syntax using `__future__`.
- Replace various ad hoc scripts with Makefile targets.
- Drop support for bootstrapping via ez_script. Use pip.
- Update the builtin STUN servers list.
- Various other code refactorings.
### Note

Some of the changes introduced are not backwards-compatible. I can remove/revert them if it's an issue.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,10 @@
+.. image:: https://travis-ci.org/Renelvon/pystun.svg?branch=master
```
Comment (jtriley) [inline README.rst]: These will reflect a different repo but I suppose that's OK for now given that you will eventually take over.

Comment (Renelvon) [inline README.rst]: Oops, my bad. I will switch them over to target your repo. Can you enable Travis CI and Coveralls on your branch?

Comment (jtriley) [inline README.rst]: Yep I'll do that today. Thanks!

**Comment by jtriley** [comment]
This looks good to me. :+1:

**Pull request merged**